### PR TITLE
get sum click for last 24 hours per app

### PR DESCRIPTION
**Pull request by shmuliko**
sum the task_queue.converted

### CODE REVIEW

File: protected/controllers/admin/apps/apps_app_controller.php [hunk 1]
```diff
@@ -91,6 +91,29 @@ public function get_closed()
...
+        $yesterdate = date('Y-m-d H:i:s', strtotime(gmdate('Y-m-d H:i:s') . ' - 1 day'));
+        $sql = ('
+            SELECT 
+                IFNULL(SUM(tq.converted), 0) converted
+            FROM
+                task_queue tq 
+                    JOIN
+                tasks t ON tq.task_id = t.id
+                    LEFT JOIN
+                sent_pushes sp ON sp.queue_id = tq.id
```
Comment (mosheka) [inline protected/controllers/admin/apps/apps_app_controller.php]: We don't need sent_pushes table anymore

Comment (mosheka) [reply protected/controllers/admin/apps/apps_app_controller.php]: And neither tasks

Comment (shmuliko) [reply protected/controllers/admin/apps/apps_app_controller.php]: u r talk as yevgeniy fix this
(because the sum is on sp table)

Comment (shmuliko) [reply protected/controllers/admin/apps/apps_app_controller.php]: i change it

**Pull request merged**